Documentation/k8s-pre-reqs.md

```diff
@@ -52,6 +52,17 @@ Rancher provides an easy way to configure Kubelet. This flag can be provided to
 
 To configure Flexvolume in Rancher, specify this Kubelet flag as shown below:
 
+```bash
+--volume-plugin-dir=/var/lib/kubelet/volumeplugins
+```
+
+Restart Kubelet in order for this change to take effect.
+
+### Atomic host
```

Comment:
Shall we merge the CoreOS, Rancher, and Atomic sections into a single section? They all use the same setting and seem to say the same thing.

Reply:
Rancher does have some extra information about how/where to add the extra argument, using the `up to date` feature.  Otherwise, we can do a refactor for the argument itself since it's the same on all platforms.